packages/core/src/llm/LLM.ts

```diff
@@ -613,10 +608,28 @@ If a question does not make any sense, or is not factually coherent, explain why
   }
 }
 
+export const ALL_AVAILABLE_ANTHROPIC_V2_MODELS = {
+  "claude-2.1": {
+    contextWindow: 200000,
+  },
+  "claude-instant-1.2": {
```

Comment:
v1.2 is not v2

Reply:
Fair point, let's call this ANTHROPIC_LEGACY_MODELS then? And add claude-2.0 for completeness.